app/views/layouts/angular/_multi_auth_credentials.html.haml

```diff
@@ -261,25 +261,48 @@
       = miq_tab_content('hawkular', 'default') do
         .form-group
           .col-md-12.col-md-12
-            = render :partial => "layouts/angular-bootstrap/endpoints_angular",
-                                 :locals  => {:ng_show                => true,
-                                              :ng_model               => "#{ng_model}",
```

Comment:
I mean https://github.com/ManageIQ/manageiq-ui-classic/pull/1205/files?w=1#diff-19776d82dcdf8bc1080fe79eb95a254bR288
there used to be `:ng_model               => "#{ng_model}",` now blank lines (?)

(not sure what it does and if removal is good, just asking)